src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -193,9 +195,10 @@ public void testActionGroupsApi() throws Exception {
         Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
 
         // PATCH read only resource, must be forbidden
+        // SuperAdmin can patch read only resource
```

Comment:
lets rename sendHTTPClientCertificate to presentAdminCertificate or sendAdminCertificate

Reply:
renamed to sendAdminCertificate